pages/common/git-stamp.md

```diff
@@ -0,0 +1,17 @@
+# git stamp
+
+> Stamp the last commit message, with the possibility to reference the issues numbers from your bug tracker or link to its review page.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-stamp>.
+
+- Stamp the last commit message referencing it with the issue number from your bug tracker:
+
+`git stamp issue {{FOO-123}}`
```

Comment:
`issue` is not part of the command, it's part of the text to be appended to the message. IMHO perhaps this better
```suggestion
`git stamp {{issue_number}}`
```

Reply:
Shouldn't it be?
```suggestion
`git stamp issue {{issue_number}}`
```

Reply:
No because the prefix `Issue` is customizable based on your repo, maybe you use `Close #FOO-21` or `#FOO-21 Close` I remember Bitbucket is in inverse order. Whatever, depends on your tracking system

Reply:
That's right. Official docs confused me, but I have tested around and its like @navarroaxel commented